lib/ansible/modules/cloud/amazon/ec2_eip_facts.py

```diff
@@ -18,8 +18,9 @@
 options:
   filters:
     description:
-      - A set of filters to use. Each filter is a name:value pair. The value
-        may be a list or a single element.
+      - A dict of filters to apply. Each dict item consists of a filter key and filter
+        value.  See https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-addresses.html#options
```

Comment:
Adding `U(...)` will make this a URL
```suggestion
        value.  See U(https://docs.aws.amazon.com/cli/latest/reference/ec2/describe-addresses.html#options)
```